it/sc/service_catalog_test.go

```diff
@@ -26,6 +27,13 @@ func TestServiceCatalog(t *testing.T) {
 				ClusterServiceClassExternalName: "user-provided-service",
 				ClusterServicePlanExternalName:  "default",
 			},
+			Parameters: &runtime.RawExtension{
+				Raw: []byte(`{
+	"credentials": {
+		"token": "token"
+	}
```

Comment:
Service Catalog not does validation of the schema and test was failing because parameters is a mandatory thing.